docs/docs/index.md

```diff
@@ -26,19 +26,17 @@ The source code is available here: <https://github.com/mundialis/actinia_core>.
 * Actinia concepts
     * Actinia REST API documentation
     * User, user-roles and user-groups
-    * The Actinia databases 
+    * The Actinia databases
 * Installation
 * Actinia database access
-    * Using curl for HTTP requests 
-    * Access to locations and mapsets in the persistent database 
-    * Access to raster layers in the persistent database 
-    * Access to raster time-series in the persistent database 
-* Time-series sampling 
-    * Sampling of a STRDS with vector points 
-* Landsat NDVI computation 
```

Comment:
Instead of deleting parts that might be added back later on, you could comment them out, e.g.
```
<!---
* Landsat NDVI computation
--->
```
That makes it easier to reactivate them later on.

Reply:
Thanks, I was not aware of this easy way of commenting in markdown! ;)